iprange compiler error after latest change

Thanks for fixing the last bug, however do you mean to also break backward compatibility? I now no longer able to compile with iprange.

```
error[E0658]: use of unstable library feature 'option_insert': newly added
   --> /home/test/.cargo/registry/src/github.com-1ecc6299db9ec823/iprange-0.6.5/src/lib.rs:435:23
    |
435 |             self.root.insert(IpTrieNode::new()) as *mut IpTrieNode
    |                       ^^^^^^
    |
    = note: see issue #78271 <https://github.com/rust-lang/rust/issues/78271> for more information

error: aborting due to previous error

For more information about this error, try `rustc --explain E0658`.
error: could not compile `iprange`
```

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -432,7 +432,8 @@ where
             }
             root as *mut IpTrieNode
         } else {
-            self.root.insert(IpTrieNode::new()) as *mut IpTrieNode
+            self.root = Some(IpTrieNode::new());
+            self.root.as_mut().unwrap() as *mut IpTrieNode
         };
 
         unsafe {

